src/d3a/d3a_core/redis_connections/redis_communication.py

```diff
@@ -63,6 +74,8 @@ def __init__(self, simulation, simulation_id, live_events):
             log.error("Redis is not operational, will not use it for communication.")
             del self.pubsub
             return
+        self.heartbeat = RepeatingTimer(PULSE_RATE, self.heartbeat_tick)
```

Comment:
I think it would be much better to name this `HEARTBEAT_INTERVAL`, it is more understandable to my eyes. Keep it that way if you disagree though